### ISSUE TITLE
Does not seem to work with pipenv

### ISSUE THREAD

**Issue by everhardt** [elarivie/linter-mypy#18]
Pipenv creates virtual environments with a directory name that cannot be derived from $PROJECT_PATH or $PROJECT_NAME. Their way to run mypy, is `pipenv run mypy ...`. or `pipenv run python -m mypy` in the project root. 

I can currently only set an executable path for python in the settings of linter-mypy. I think this issue would therefore be resolved if I could provide an executable path plus arguments (eg. `$PROJECT_PATH/pipenv run python`).

**Comment by elarivie** [comment]
Hello @everhardt 

I am not an expert of pipenv but I see two options

**Option 1.**  Call ["pipenv shell"](https://pipenv.readthedocs.io/en/latest/) which will adjust the environment variable to provide access to the python interpreter of the current virtual environment and then launch atom within that context.
You will then be able to configure linter-mypy by simply providing the interpreter name **python**

**Option 2.**  Use a wrapper script like for example:


[pipenv_python_wrapper.sh](https://github.com/elarivie/linter-mypy/files/3033807/pipenv_python_wrapper.txt)

**Note:** Put the wrapper file at the root of your project (next to the Pipfile).
You will then be able to configure linter-mypy by simply providing the interpreter **$PROJECT_PATH/[pipenv_python_wrapper.sh](https://github.com/elarivie/linter-mypy/files/3033807/pipenv_python_wrapper.txt)**

Let me know if those options solve your issue.

**Comment by everhardt** [comment]
Thanks for your quick reply! I forgot to mention that I already start atom with `pipenv run atom .`. linter-mypy does give valid results, but fails to find imported modules: `Cannot find module named 'xxx'`, which is why I assumed it didn't play nice with pipenv.

I did some debugging in the code of linter-mypy and noticed that I get those errors if I run the command `executablePath + " " + params.join(" ")` from `/` (as `options.cwd` stated). With that, it was easy to derive that I missed setting MYPYPATH manually on top of that. So in the end, I fixed this by:
* launching atom from the command line with `pipenv run atom .` (I guess second suggestion would also work)
* setting the MYPYPATH to `$PROJECT_PATH`
